Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DFSAffineTransform.txx

```diff
@@ -28,15 +28,15 @@ DiffusionTensor3DFSAffineTransform<TData>
 
   vnl_matrix_fixed<double, 3, 3> M( 3, 3 );
   M = matrix.GetVnlMatrix();
-  vnl_real_eigensystem             eig( M );
+  vnl_real_eigensystem             eig( M.as_ref() );
```

Comment:
@hjmjohnson  Would appreciate if you could have a quick look at this patch and also this one https://github.com/Slicer/BRAINSTools/commit/d87ea8f357622c27abc6a2162b4623abe782b995

Reply:
@jcfr This looks fine.  If you can match types, there can be some efficiencies using move constructors.  For a case like this, it probably does not make any significant difference using  `as_ref()` vs `as_matrix()`.

Hans

Reply:
Thanks for the review.

If recall, I tried to use `.as_matrix()` and it failed. I re-compiling locally and will post an update here.